run-tests.js

```diff
@@ -213,14 +213,30 @@ const sequentialMap = async (input, fn) => {
 
 const testExchange = async (exchange) => {
 
-    numExchangesTested++
-    const percentsDone = ((numExchangesTested / exchanges.length) * 100).toFixed (0) + '%'
+    const percentsDone = () => ((numExchangesTested / exchanges.length) * 100).toFixed (0) + '%';
```

Comment:
we should't increase counter during async method start, because it gives unsorted results during tests, like:
```
94% done
82% done
96% done...
```
we'd better increase counter once the test for exchange is complete (so, the variable is moved below)